cleedpy/cleed/src/lpcmktlnd.c

```diff
@@ -101,7 +101,7 @@ struct phs_str *ptr;
 
  if(p_tl == NULL)
  {
-   p_tl = (mat *)malloc(n_set * sizeof(mat));
+   p_tl = (mat *)calloc(n_set, sizeof(mat));
```

Comment:
please change back to `malloc`

Reply:
done in [86589e4](https://github.com/empa-scientific-it/cleedpy/pull/49/commits/86589e4b1ea9e7533da79fef6e2443dcd799741e)